interpreter/cling/lib/Interpreter/ValuePrinter.cpp

```diff
@@ -433,23 +189,31 @@ static std::string getCastValueString(const Value &V) {
     typeWithOptDeref << "(" << type << ")";
   } else if (Ty->isPointerType()) {
     if (Ty->getPointeeType()->isCharType()) {
+      // Print char pointers as strings.
       typeWithOptDeref << "(" << type << ")";
```

Comment:
That is mighty dangerous. This of the following `char* = &gCling` - I have no idea where that "string" ends and I am fairly sure that my terminal will be unhappy seeing this :-)